ansible/roles/candlepin/tasks/el9/el9_ruby.yml

```diff
@@ -43,6 +43,6 @@
     - ruby
 
 - name: Ensure bundler is installed
-  ansible.builtin.command: "bash -lc 'gem install bundler'"
+  ansible.builtin.command: "bash -lc 'gem install bundler -v 2.3.26'"
```

Comment:
Is this an issue just in rhel9, or does it also need to be set in other OSes?

Reply:
Good question. RVM role is not smart enough to deal with this so I updated EL8 as well.